File: Meditation/app/src/main/java/com/example/meditation/HomeActivity.java
package com.example.meditation;

import android.content.Intent;
import android.os.Bundle;

import androidx.appcompat.app.AppCompatActivity;

import com.example.meditation.databinding.ActivityHomeBinding;

import java.time.Instant;

public class HomeActivity extends AppCompatActivity {
    
    ActivityHomeBinding homeBinding;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        homeBinding=ActivityHomeBinding.inflate(getLayoutInflater());
        setContentView(homeBinding.getRoot());

        view();
    }

    private void view() {
        homeBinding.cdYoga.setOnClickListener(y->{
            Intent yoga=new Intent(this,YogaActivity.class);
            startActivity(yoga);
        });
        homeBinding.cdMeditation.setOnClickListener(me->{
            Intent meditation=new Intent(this,MeditationActivity.class);
            startActivity(meditation);


        });
//        homeBinding.cdMusic.setOnClickListener(mu->{
//            Intent music=new Intent(this,MusicActivity.class);
//            startActivity(music);
//        });
        homeBinding.cdMantra.setOnClickListener(mu->{
            Intent mantra=new Intent(this,MantraActivity.class);
            startActivity(mantra);
        });

    }
}

File: Pro1/app/src/main/java/com/example/pro1/activity/HomeActivity.kt
package com.example.pro1.activity

import android.content.Intent
import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.example.pro1.databinding.ActivityHomeBinding
import com.example.pro1.databinding.AddDetailsListBinding


class HomeActivity : AppCompatActivity() {


    lateinit var homeBinding: ActivityHomeBinding


    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        homeBinding = ActivityHomeBinding.inflate(layoutInflater)
        setContentView(homeBinding.root)

initview()
    }

    private fun initview() {

        homeBinding.cdSalesDetails.setOnClickListener{
            var sales=Intent(this,SalesDetailsActivity::class.java)
            startActivity(sales)
        }
        homeBinding.cdCategory.setOnClickListener{
            var i=Intent(this,CategoryListActivity::class.java)
            startActivity(i)
        }

        homeBinding.linAdd.setOnClickListener{
            var newBill=Intent(this,AddDetailsActivity::class.java)
            startActivity(newBill)
        }

    }


}